WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Warning/RestoreWarningView.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+import Gridicons
+import WordPressUI
+
+class RestoreWarningView: UIView, NibLoadable {
+
+    // MARK: - Properties
+
+    @IBOutlet private weak var icon: UIImageView!
+    @IBOutlet private weak var title: UILabel!
+    @IBOutlet private weak var body: UILabel!
+    @IBOutlet private weak var confirmButton: UIButton!
+    @IBOutlet private weak var cancelButton: UIButton!
+
+    var confirmHandler: (() -> Void)?
+    var cancelHandler: (() -> Void)?
+
+    // MARK: - Initialization
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        applyStyles()
+        configure()
+    }
+
+    // MARK: - Styling
+
+    private func applyStyles() {
+        backgroundColor = .white
```

Comment:
For light/dark mode:

```suggestion
        backgroundColor = .basicBackground
```